docs/source/run/parameters.rst

```diff
@@ -38,6 +38,88 @@ as command line parameters. Multi-line expressions are allowed if surrounded by
 General parameters
 ------------------
 
+* ``hipace.normalized_units`` (`bool`) optional (default `0`)
+    Using normalized units in the simulation.
+
+* ``random_seed`` (`integer`) optional (default `1`)
+    Passes a seed to the AMReX random number generator. This allows for reproducibility of random events such as randomly generated beams, ionization, and collisions.
+    Note that on GPU, since the order of operations is not ensured, the providing of a seed does not guarantee reproducibility to the level of machine precision.
```

Comment:
```suggestion
    Note that on GPU, since the order of operations is not ensured, providing a seed does not guarantee reproducibility to the level of machine precision.
```